jira/client.py

```diff
@@ -3983,6 +4042,17 @@ def permissionschemes(self):
 
         return data["permissionSchemes"]
 
+    @lru_cache(maxsize=None)
+    def issue_type_schemes(self) -> List[IssueTypeScheme]:
+        """get all issue type schemes defined"""
```

Comment:
This needs the docstring in the Google style

```suggestion
        """Get all issue type schemes defined (Admin required).
        
        Returns:
            List[IssueTypeScheme]: All the Issue Type Schemes available to the currently logged in user.
        """
```